ansible_builder/main.py

```diff
@@ -98,17 +103,15 @@ def __init__(self, filename):
             raise DefinitionError("""
             Could not detect '{0}' file in this directory.
             Use -f to specify a different location.
-            """.format(constants.default_file))
+            """.format(filename))
```

Comment:
Ah, is this the reason that it wasn't printing the right filename?

Reply:
Yep!  It was always filling that line with the default name vs what was specified (_if_ something was specified).